src/MoBi.Engine/Sbml/SBMLTask.cs

```diff
@@ -104,9 +104,14 @@ public Model GetModel(string filename)
          {
             throw new MoBiException(SBMLConstants.ModelNotRead(sbmlDoc.getErrorLog().ToString()));
          }
-
-         convertSBML(sbmlDoc);
-
+         try
+         {
+            convertSBML(sbmlDoc);
+         }
+         catch (Exception _)
```

Comment:
Are we sure this is always the reason for an exception? Cannot we be more explicit here? What if the code in `convertSBML` (our code) crashes?